www/index.html.pm

```diff
@@ -296,12 +242,8 @@ ancestor(A, B)?}}}
 
      ◊p[#:class "block-with-1em-margin"]{
         While Racket is a functional language, it has offered a sub-language of ◊link[traits-aplas]{classes and objects, mixins and traits}, from the beginning. The macro-based implementation of a Java-like class system lives in a library and does not need any support from the core language. A Racket programmer can thus combine functional with object-oriented components as needed. }}
-
-}}}
-
-◊div[#:id "hash-langs" #:class "lop full-mode-menu-content unselected-block"]{
- ◊special-section[#:class "one-column-body-text"]{
-
+}}
+  ◊tab[#:id "hash-langs" #:title "Easy DSLs"]{
```

Comment:
A future PR will add a support of anchor linking (e.g. `https://www.racket-lang.org/index.html#hash-langs`), so it would be great to review that these ids all make sense.